OpenRA.Mods.Common/MapGenerator/ActorPlan.cs

```diff
@@ -103,6 +103,17 @@ public IReadOnlyDictionary<CPos, SubCell> Footprint()
 				return occupiedCells;
 		}
 
+		/// <summary>
+		/// Return true iff any of the actors footprint satisfies Map.Contains.
```

Comment:
Comment & commit says "any of the actors footprint satisfies" but implementation is `.All`. I think all is likely correct - no use is the actor is hanging partially off the map - so just flagging in case the comment has become outdated.